### ISSUE TITLE
Evaluate code in Magento not copyable

### ISSUE THREAD

**Issue by Schrank** [enriquepiatti/Magicento#121]
When code in Magento is evaluated, the var_export result is shown in a popup - but I can't copy the content :( Would be great if you find a form, where I can copy from, textfield, copyable text?

![bildschirmfoto 2014-04-02 um 16 04 49](https://cloud.githubusercontent.com/assets/379680/2591854/cc600bc2-ba6f-11e3-85e4-fe896036244b.png)

Thanks! :)

**Comment by enriquepiatti** [comment]
Hi Fabian, what OS are you using? I can copy it on Windows, not with right click, but I can select the text and copy it with Ctrl+C. 
Is not possible to do that in your OS (I have not tested that with Linux and I don't have a Mac :( )

**Comment by Schrank** [comment]
No, it is not possible on Mac OS X :-( No selection, no right click. I was a little bit suprised, because I was sure this worked in the past... Maybe it is EAP 8 :D I'll ask around for more input.
